aerospike/test_aerospike.py

```diff
@@ -22,17 +22,25 @@
 
 @attr(requires='aerospike')
 class TestAerospike(AgentCheckTest):
-    """Basic Test for aerospike integration."""
     CHECK_NAME = 'aerospike'
 
     def test_check(self):
-        """
-        Testing Aerospike check.
-        """
-        self.load_check({}, {})
+        config = {
+            'instances': [
+                {
+                    'metrics':['cluster_size'],
+                    'namespace_metrics':['objects','hwm_breached'],
+                    'namespaces':{'test'},
+                    'tags':['tag:value']
+                }
+            ]
+        }
+        self.run_check(config)
+
+        self.assertMetric('aerospike.cluster_size', at_least=1)
```

Comment:
If we move the metric catalog to the check, it should be fairly straight-forward to iterate over some of the metrics to get some better metric catalog coverage? Would that be something you could entertain? I know I'm being greedy here, great work :)